Makefile

```diff
@@ -90,6 +93,10 @@ ifneq ($(wildcard /.dockerenv), )
 	go test -check.v -check.timeout=$(TEST_TIMEOUT) timeout=$(TEST_SUITE_TIMEOUT) $(TESTFLAGS) github.com/docker/containerd/integration-test
```

Comment:
since we're going to do it eventually, we might as well do it now I'd say, it's a trivial change :)